Source/WebCore/platform/graphics/cg/ImageDecoderCG.h

```diff
@@ -50,7 +50,7 @@ class ImageDecoderCG final : public ImageDecoder {
     IntSize size() const final { return IntSize(); }
     size_t frameCount() const final;
     RepetitionCount repetitionCount() const final;
-    String uti() const final;
+    String uti() const final { return m_uti; }
```

Comment:
Is it possible to add a test? If we know which encoder creates the problematic files, it seems like we could make a test image and commit it.

Reply:
I added a ref test which is enabled only on macOS Ventura only for WK1. It is marked failed on WK2 because of the WKR sandboxing issue webkit.org/b/240987.